### DISCUSSION TITLE
Additional users - can't see links

### DISCUSSION THREAD

**Discussion by jvanas** [Kovah/LinkAce#582] [Support]
I am hoping to use LinkAce to share links with a team.  I created a second user - he can login.  But he can't see any of the links I already made.   When he creates a link, I can't see it.  None of the links are marked as private.  Guest link sharing is enabled in System Settings.  Does LinkAce support link sharing between users?  Thanks!

**Comment by Kovah** [comment]
Hi. At the moment LinkAce does not support multiple users at all. Sorry.

If you want to use more than one user, follow this issue: 

- #165
